src/coffee/bootstrap-tour.spec.coffee

```diff
@@ -739,11 +739,23 @@ describe 'Bootstrap Tour', ->
       path: 'test.html'
       host: 'http://sub.exemple.com'
 
+    @tour.addStep
+      element: $('<div></div>').appendTo('body')
+      path: 'test.html'
+      host: /exemple\.com$/
```

Comment:
`example.com` please instead of `exemple.com` (which is french). :P Same everywhere else.